Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -36,6 +36,8 @@ def linkage
             libgcc_s.so.1
             libgomp.so.1
             libstdc++.so.6
+
+            ld-linux-x86-64.so.2
```

Comment:
Ah, I didn't realize that this was actually part of `glibc`. I thought it was part of the kernel.

Thanks!